src/conduits/RwaOutputConduit3.sol

```diff
@@ -278,7 +278,7 @@ contract RwaOutputConduit3 {
      * @notice Method to swap DAI contract balance to GEM through PSM and push it to the recipient address.
      * @dev `msg.sender` must have been `mate`d and `to` must have been `pick`ed.
      */
-    function push() external isMate {
+    function push() external isMate { // TB - consider adding a slippage check here and in the other push function for the output, to protect against unexpected fee changes. I.e add a parameter of minGemAmount. It's pretty common for swapping logic.
```

Comment:
Slippage protection seems like an overkill here, since the fees can only be changed by governance, with a very well defined schedule and this function will rarely be called. The chances of `push()` being "front-runned" with a spell execution are pretty slim.